src/Illuminate/Database/Eloquent/Builder.php

```diff
@@ -602,7 +602,7 @@ public function valueOrFail($column)
      */
     public function get($columns = ['*'])
     {
-        $builder = $this->applyScopes();
+        $builder = $this->applyOrderBy()->applyScopes();
```

Comment:
What's if order by is applied via scope, and based on the code scope only applied after order by?

Reply:
@crynobone That's a very good point. Maybe @jasonmccreary these chained calls need to be the other way around?

Reply:
@crynobone @jasonmccreary Fix for the method ordering committed to my fork https://github.com/jonnott/framework/commit/b835133c682a57ded5cf70029c76a0e46ba41b86

Reply:
@jasonmccreary I've also added a test for @crynobone's mentioned situation where an orderBy() is applied via a scope: https://github.com/jonnott/framework/commit/231d4f6aacc8896c63ccdcf392dd93996aca9196 (test would've failed without the switching of `applyDefaultOrderBy()->applyScopes()` to `applyScopes()->applyDefaultOrderBy()`)